hydrogen-production-pres/conclusions.tex

```diff
@@ -4,8 +4,8 @@
 \begin{itemize}
 	\item The University of Illinois is actively working to reduce GHG emissions on its campus. 
 	\item A few microreactor designs would be able to produce enough hydrogen to meet MTD and UIUC fleet fuel demand.
-	\item An excessive integration of PV to UIUC grid make the duck curve phenomenon likely to occur.
-	\item Nuclear energy and hydrogen production proposes an approach to mitigate the negative implications of the duck curve.
+	\item An excessive integration of PV to UIUC grid will make the duck curve phenomenon likely to occur.
```

Comment:
```suggestion
	\item Increased solar penetration worsens the duck curve.
```

Reply:
Not essential, just a suggestion for clarity.

Reply:
looks good